structure/views.py

```diff
@@ -489,7 +489,8 @@ def get_context_data (self, **kwargs):
 		context['class_f_options']['label_free'] = [1,]
 		#json.dump(class_f_data.get_nodes_dict('crystalized'), open('tree_test.json', 'w'), indent=4)
 		context['class_f'] = json.dumps(class_f_data.get_nodes_dict('crystals'))
-		class_t2_data = tree.get_tree_data(ProteinFamily.objects.get(name='Class T (Taste 2)'))
+                class_t2_data = tree.get_tree_data(ProteinFamily.objects.get(name='Class T (Taste 2)'))

```

Comment:
Doesn't the newly introduced indentation break the execution?
The commented line below also doesn't have to be added imho.

Reply:
You're right.
Fixing.